test/test_jit.py

```diff
@@ -748,6 +748,14 @@ def fn(*args):
                 self.assertFalse(fn.has_trace_for(*unk_config))
         self.assertEqual(fn.hits, 0)
 
+    def test_optional_args(self):
+        @torch.jit.script
+        def norm_test():
```

Comment:
Instead of adding it here, can you add a entry to https://github.com/pytorch/pytorch/blob/master/test/common_methods_invocations.py#L524? JIT will automatically pick tests there